cmd/livepeer/livepeer.go

```diff
@@ -815,6 +864,9 @@ func main() {
 	case err := <-ec:
 		glog.Infof("Error from media server: %v", err)
 		return
+	case err := <-redeemerErr:
+		glog.Errorf("Error starting redemption service: %v", err)
```

Comment:
Since `grpc.Server.Serve()` (called by `RedeemerClient.Start()`) will return nil if `grpc.Server.Stop()` is called, maybe we should check if the error is not nil here before logging.

Reply:
fixed in c0f617298df0d7c9472e110eb0d33e06f5cc99bf